client/app/components/sections/ManageAvailability/ManageAvailability.js

```diff
@@ -37,14 +39,23 @@ const shouldLoad = (isOpen, prevIsOpen) => isOpen && !prevIsOpen;
    component `isOpen` becomes `true`
 */
 const loadInitialDataIfNeeded = (props, prevProps = null) => {
-  const isOpen = props.winder.isOpen;
-  const prevIsOpen = prevProps && prevProps.winder.isOpen;
+  const isOpen = props.isOpen;
+  const prevIsOpen = prevProps && prevProps.isOpen;
 
   if (shouldLoad(isOpen, prevIsOpen)) {
     props.calendar.onMonthChanged(props.calendar.initialMonth);
   }
 };
 
+const setPushState = (state, title, path) => {
```

Comment:
If the Storybook works, it should be fine. The component is rendered in an iframe, so the url it manipulates would be the inner frame url and doesn't mess with the URL of the Storybook itself.